### PR TITLE
Add svg logo for edX, and remove siteMetadata

### PR DESCRIPTION
**Pull request by adamstankiewicz**
The header and footer for "Master's Home" should have the edX logo, rather than being one of the configurable things during build time. This PR does the following:
1. Adds an SVG edX logo, updates the `Layout` component to use the logo in the `SiteHeader` and `SiteFooter`.
2. Because this image and site name are always going to be edX specific, I removed the `siteName` and `logo` from `siteMetadata` in `gatsby-config.js` as well as the `LayoutQuery` as that component no longer has a need to get that data from the Gatsby config. We can always add it back if/when we have a need for it.

### CODE REVIEW

**Comment by adamstankiewicz** [comment]
@tuchfarber Pushed up another commit to address your feedback. Basically, if a `logo` is supplied in `gatsby-config.js`, use that; otherwise, use the edX SVG logo.

**Comment by tuchfarber** [comment]
I'd still lean towards having the default be an Open edX logo like the rest of our applications. If we need this for right now, I won't block it, but when we enable builds able to take this as configuration, the Open edX logo should become the default, not the edX logo.

**Comment by adamstankiewicz** [comment]
@tuchfarber Ah, gotcha. I will update. Thanks!

**Comment by erinakii** [comment]
@tuchfarber @adamstankiewicz do you have thoughts on the best place to set edxLogo globally? In env.development and add to whitelist in gatsby.config?

File: jest.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
   },
   moduleNameMapper: {
     '.+\\.(css|styl|less|sass|scss)$': 'identity-obj-proxy',
-    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/__mocks__/file-mock.js',
+    '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/fileTransformer.js',
```
Comment (erinakii) [inline jest.config.js]: using _mocks_/file-mock.js doesn't seem to work here (or potentially exist?) with the svg file of the edx logo. It thew a configuration error when running tests in programListPage. I added the fileTransformer file which seems to work a little better. I made this change for testing image files based on jest documentation that I found worked. 

https://jestjs.io/docs/en/webpack

**Pull request closed without merging**